packages/cloud_firestore/cloud_firestore_web/lib/src/load_bundle_task_web.dart

```diff
@@ -5,29 +5,16 @@ import 'interop/firestore.dart';
 
 class LoadBundleTaskWeb extends LoadBundleTaskPlatform {
   LoadBundleTaskWeb(LoadBundleTask task) : super() {
-    Stream<LoadBundleTaskSnapshotPlatform> mapNativeStream() async* {
-      await for (final snapshot in task.stream()) {
-        Map<String, dynamic> data = {
-          'bytesLoaded': snapshot.bytesLoaded,
-          'documentsLoaded': snapshot.documentsLoaded,
-          'totalBytes': snapshot.totalBytes,
-          'totalDocuments': snapshot.totalDocuments
-        };
+    stream = task.stream().map((snapshot) {
```

Comment:
This isn't a broadcast stream anymore. I don't think that's voluntary

Reply:
I don't understand what is required? 😅

Reply:
It's missing an `asBroadcastStream` (with a pause/resume we discussed before too)

And possibly a test for that